backend/QA/src/main/java/com/example/qa/dao/question/QuestionDao.java

```diff
@@ -188,6 +188,14 @@ public void update(
       throws DaoException {
     Put put = createPutWithForUpdate(date, createdAt, updatedAt, numAnswer);
 
+    /** Read the updating record and update all values for a workaround of the partial update
+     * because Cosmos DB does not support partial updating.
+     * For other data stores, this is not needed. */
+    QuestionRecord question = get(date, createdAt, storage);
+    put.withValue(new TextValue(COL_NAME_TITLE, question.getTitle()))

```

Comment:
@yito88 
I think it should be taken cared by the library side until it is fixed.
Otherwise, it might be too error-prone.
What do you think?

Reply:
@feeblefakie Thank you for the suggestion.
OK, I'll modify the put process of Cosmos DB adaptor in Scalar DB.

@scalar-boney Sorry for the inconvenience. Could you revert the modifications?

Reply:
@yito88 san,
Thank you for your confirmation.

> Could you revert the modifications?

Cosmos DB specific update logic reverted.